acquisitions/src/main/resources/thunderjet/mod-orders/features/routing-list-print-template.feature

```diff
@@ -0,0 +1,161 @@
+      And match response.personal.addresses[*].addressLine1 contains officeAddressLine1V1
+      And match response.personal.addresses[*].addressTypeId contains homeAddressTypeId
+      And match response.personal.addresses[*].addressLine1 contains homeAddressLine1V1
+
+      Given path 'users'
+      And request user2
+      When method POST
+      Then status 201
+      And match response.personal.addresses[*].addressTypeId contains officeAddressTypeId
+      And match response.personal.addresses[*].addressLine1 contains officeAddressLine1V2
+      And match response.personal.addresses[*].addressTypeId contains homeAddressTypeId
+      And match response.personal.addresses[*].addressLine1 contains homeAddressLine1V2
+
+    Scenario: Create composite order to use in routing list
+      Given path 'orders/composite-orders'
+      And request
+        """
+        {
+          id: '#(orderId)',
+          vendor: '#(globalVendorId)',
+          orderType: 'One-Time'
+        }
+        """
+      When method POST
+      Then status 201
+
+    Scenario: Create order line for composite order that will be used in routingList
+      Given path 'orders/order-lines'
+
+      * def orderLine = read('classpath:samples/mod-orders/orderLines/minimal-order-line.json')
+      * set orderLine.id = poLineId
+      * set orderLine.purchaseOrderId = orderId
+
+      And request orderLine
+      When method POST
+      Then status 201
+      * def response = $
+      And match response.paymentStatus == "Pending"
+
+    Scenario: Create Routing Lists with two user 'user1', 'user2'
+      * def routingList = read('classpath:samples/mod-orders/routingLists/' + routingListId + '.json')
+      * set routingList.id = routingListId
+      * set routingList.poLineId = poLineId
+      * configure headers = headersAdmin
+
+      Given path 'orders-storage/routing-lists'
+      And request routingList
+      When method POST
+      Then status 201
+      And match response.id == routingListId
+
+    Scenario: POST setting with addressTypId
+      * configure headers = headersAdmin
+      Given path 'orders-storage/settings'
+      And request
+        """
+        {
+          "key": "ROUTING_USER_ADDRESS_TYPE_ID",
+          "value": "#(officeAddressTypeId)"
+        }
+        """
+      When method POST
+      Then status 201
+      And match response.key == 'ROUTING_USER_ADDRESS_TYPE_ID'
+      And match response.value == officeAddressTypeId
+
+    Scenario: POST template config in mod-template-engine to use
+      * configure headers = headersAdmin
+      * def templateRequest = read('classpath:samples/template-config.json')
+      * set templateRequest.id = templateId
+      Given path 'templates'
+      And request templateRequest
+      When method POST
+      Then status 201
+      And match response.id == templateId
+
+
+    Scenario: Send request to print routing lists
+      * configure headers = headersAdmin
```

Comment:
this part changed to headersUser after https://github.com/folio-org/mod-orders/pull/879 pr merged